content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -1141,7 +1068,7 @@ For example, the following query returns `Post` records that meet the following
 - Author's name is not Bob
 - Author is older than 40
 
-```ts
+```ts highlight=3-13;normal
```

Comment:
glitchy indentation (the first line has two leading spaces):

<img width="522" alt="image" src="https://github.com/prisma/docs/assets/4058327/2877cf2c-f75d-4402-9689-3e3c59828ba9">